monkey/monkey_island/cc/setup/island_config_options_validator.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from common.utils.exceptions import InsecurePermissionsError
+from monkey_island.cc.server_utils.file_utils import has_expected_permissions
+from monkey_island.cc.setup.island_config_options import IslandConfigOptions
+
+
+def raise_on_invalid_options(options: IslandConfigOptions):
```

Comment:
I think this should be named to something that conveys that it's checking only crt_path and key_path.